playbooks/common/openshift-node/image_prep.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I like this one.

On a side note for documentation, we do not use the `std_include.yml` here since it also includes `openshift_version`.  `openshift_version` requires us to have a master installed to verify that we match versions.  This is problematic as we do not have a cluster or even a master at this time.